app/subscriber/src/components/share-menu/ShareMenu.tsx

```diff
@@ -141,7 +145,10 @@ export const ShareMenu: React.FC<IShareSubMenuProps> = ({ content }) => {
         type="default"
         confirmText="Share"
         enableConfirm={
-          content.length > 0 && (user?.colleague?.email !== undefined || emailAddress !== '')
+          content.length > 0 &&
+          (user?.colleague?.email !== undefined ||
```

Comment:
Our models prefer to return empty string rather than null values most of the time. 
This will not handle an empty string correctly

Reply:
`email` and `preferredEmail` will never be `undefined` if the colleague exists.